connect-button/src/main/java/com/ifttt/connect/ui/ConnectButton.java

```diff
@@ -96,6 +98,20 @@ public boolean onPreDraw() {
      * @param configuration Configuration object that helps set up the Connect Button.
      */
     public void setup(Configuration configuration) {
+        setup(configuration, false);
+    }
+
+    /**
+     * Set up the Connect Button to fetch the Connection data with the given id and set up the View to be able to do
+     * authentication.
+     *
+     * @param configuration Configuration object that helps set up the Connect Button.
+     * @param analyticsOptOut boolean to allow opting out of analytics. If set to true, analytics event tracking will be disabled.
+     */
+    public void setup(Configuration configuration, boolean analyticsOptOut) {
```

Comment:
I would say we should add a field in the `Configuration` class that has `disableAnalyticsTracking` method. Once called, we will set the flag within the Configuration. I think it aligns better with what we have now.